Assets/HoloToolkit/Utilities/Scripts/FileSystemHelper.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if UNITY_EDITOR
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    public static class FileSystemHelper
```

Comment:
Class headers? Sure thing.

The readme in the utilities folder is pretty broken - it's tracking issues instead of using actual issues, it's really hard to read, tons of typos, duplicates information from class headers but is updated less frequently, etc.  I'd suggest instead that we switch to the more common model of only having a readme in the root.  The current system seems unmaintainable.